libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -131,6 +131,7 @@ singleton AP_Notify method handle_rgb void uint8_t 0 UINT8_MAX uint8_t 0 UINT8_M
 
 include AP_Proximity/AP_Proximity.h
```

Comment:
You can depends the include too. 
```suggestion
include AP_Proximity/AP_Proximity.h depends HAL_PROXIMITY_ENABLED == 1
```

Probably not important since there is nothing in the file if not `HAL_PROXIMITY_ENABLED`, but might save a few bytes. Although I notice we didn't do that for terrain for some reason.

Reply:
the HAL_PROXIMITY_ENABLED is set from within AP_Proximity.h so I think the way it is is probably correct.  txs for the review.